tsconfig.json

```diff
@@ -23,16 +22,5 @@
         "paths": {
             "@agm/*": ["./packages/*"]
         }
-    },
```

Comment:
are these changes necessary for universal to work?

Reply:
I don't think so, will remove